index.js

```diff
@@ -728,6 +727,75 @@ module.exports = {
     })
   },
 
+  /**
+   * Gets a statistic on one field at a time
+   * Supports where and geometry filters and group by
+   * @param {String} table to get data from
+   * @param {String} field to generate stats from
```

Comment:
shouldn't we take multiple fields and stats?

otherwise to get max and min oid, you'd have to make two calls.